expes/expe_fig_cross_val/main.py

```diff
@@ -44,7 +44,7 @@
 # grad_search
 model = Lasso(X_train, y_train, np.log(alpha_max/10))
 criterion = CV(X_val, y_val, model, X_test=X_test, y_test=y_test)
-algo = ImplicitForward(criterion, use_sk=True)
```

Comment:
before you were using sklearn but now not as you do not pass estimator to model.

on purpose?